config.json

```diff
@@ -34,7 +34,7 @@
       "description": "Specifies the distro rid for Unix OS.",
       "valueType": "property",
       "values": [],
-      "defaultValue": "${OSRid}"
+      "defaultValue": "${OSRid}-${CPUArch}"
```

Comment:
It's an intrinsic that run understands:

https://github.com/dotnet/buildtools/blob/78260239e4db68ad5c2df774a27d5f9a54fd92e5/src/Run/SettingValueProvider.cs#L72
https://github.com/dotnet/buildtools/blob/78260239e4db68ad5c2df774a27d5f9a54fd92e5/src/Run/Interop.cs#L12

Ideally it would be using M.N.PA but here we are.